### ISSUE TITLE
MRtrix3 on WSL - Qt5 issues

### ISSUE THREAD

**Issue by jdtournier** [MRtrix3/mrtrix3#2048]
Just had a go at verifying operation on WSL, using Ubuntu 20.04, and came across this issue. I thought I'd document this here is case it helps anyone else. 

Problem is that `configure` fails at the Qt check with this message in the log:
```
error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
```
Turns out this is not an unknown problem, and far from trivial to fix - see [discussion on the Microsoft WSL issue tracker](https://github.com/microsoft/WSL/issues/3023). It can be made to work using their "atomic bomb" fix - basically (as root):
```
find /lib /usr/lib /usr/libexec -name '*.so' | xargs strip --remove-section=.note.ABI-tag
```
but ultimately it's a WSL bug. Hopefully it'll magically fix itself...

**Comment by jdtournier** [comment]
OK, just to update on the graphics issue, everything seems to work fine with WSL, Ubuntu 20.04 (with fix above) using [mobaxterm](https://mobaxterm.mobatek.net/) - but not [Xming](http://www.straightrunning.com/XmingNotes/).

**Comment by bjeurissen** [comment]
Cool. How is the performance? Have you tried https://x410.dev/ before?

**Comment by bjeurissen** [comment]
> Have you tried https://x410.dev/ before?
I just realized it is not free...

**Comment by jdtournier** [comment]
Good catch. I've just tried it with the trial version, it works very well - looks better than with mobaxterm. But it's not cheap - even if they do have an 80% off offer at the moment...

**Comment by bjeurissen** [comment]
There is also https://sourceforge.net/projects/vcxsrv/

**Comment by jdtournier** [comment]
> How is the performance? 

Remarkably good given it's all software rendering - on both mobaxterm and x410. Usable frame rates when displaying ODFs, and not too bad when displaying a 5,000 streamline tractogram with lighting enabled. 

Volume rendering a T1 didn't work though, either using x410 or mobaxterm... Same dataset worked fine via MSYS2 though.

**Comment by jdtournier** [comment]
> There is also https://sourceforge.net/projects/vcxsrv/

OK, that also works. The MRView window looks the same as with mobaxterm, but the interface looks identical to XMing. It only work if 'native OpenGL' is disabled though (though still has to use software rendering). Still no volume render though.

**Comment by jdtournier** [comment]
OK, based on that, I'm happy to add a [wiki entry](https://community.mrtrix.org/c/wiki/) on setting up MRtrix3 via WSL - but until everything actually works, I don't think it should be listed as a recommended method of installation, either on the [main docs](https://mrtrix.readthedocs.io/en/latest/installation/build_from_source.html) or the [mrtrix.org downloads portal](https://www.mrtrix.org/download/)...
